.travis.yml

```diff
@@ -13,12 +14,16 @@ matrix:
     env: TASK=BUILD
     os: linux
 
+  - env: TASK=CHECK_GIT_HISTORY
+    os: linux
+
   # Work around https://github.com/travis-ci/travis-ci/issues/2317
   - env: TASK=BUILD
     os: osx
 
-  - env: TASK=CHECK_GIT_HISTORY
-    os: linux
+  allow_failures:
+  - env: TASK=BUILD
+    os: osx
```

Comment:
Could you please add a comment explaining that it is only an allowed failure due to the slow builds?